Amlogic: Enable Crazycat, Hauppauge dvb driver addon

Raybuntu opened the pull request:
PR makes it possible to build the crazycat for AML devices. Currently only C2 and WH are enabled.
WP2 is working with my builds too but it will require a new precompiled wetekdvb blobs.

All the required AML hacks have been moved to crazycat/patches/amlogic/driver.dvb.crazycat-04-AML-hacks.patch
The PR also includes a linux package bump from https://github.com/LibreELEC/LibreELEC.tv/pull/2404. I will drop it once 2404 is merged.

CONFIG_VIDEO_TVP5150 and CONFIG_DVB_LGDT3306A had to be disabled. There is probably a better way to blacklist those on AML. ping @CvH

Raybuntu commented:
OK I've tested recent changes. @CvH reworked most of the commits and added Hauppage support. I've added WIP label cause I wanna wait for https://github.com/LibreELEC/LibreELEC.tv/pull/2421. Then I'll rebase add CC and Hauppage support to Amlogic project.
Meson-IR patch needs fixup to include recent fixes by @HiassofT.
@afl1 FYI.

afl1 commented on projects/Amlogic/options:
```diff
@@ -143,3 +143,11 @@
+  # driver addons to install:
+  # for a list of additinoal drivers see packages/linux-driver-addons
+  # Space separated list is supported,
+    DRIVER_ADDONS="crazycat hauppauge"
```
We are building them as a LE repository addons. Update for these  addons is less frequent then LE updates. Also we can save about 9MB in install image.

Raybuntu replied:
It has pros and cons. What happens if we bump the kernel?

CvH replied:
@afl1 that approach only works without hassle at AML as there are no kernel updates, every other project would fail without nasty workarounds at up/downgrade scenarios

chewitt replied:
The only image that has size constraints is Generic (x86_64) where we need to be aware of older OE installs with 235MB boot partitions. All other build projects are a mile clear of that restriction and we should embed drivers into the image to avoid mismatched driver issues from occurring. Bandwidth is cheap. Human effort debugging driver issues is not.

Raybuntu commented:
I removed WH since this will work ootb once https://github.com/LibreELEC/LibreELEC.tv/pull/2444 is merged. I've also rebased the meson-ir patch for both hauppauge and crazycat to include @HiassofT fixes for ghost presses with ir-keytable.
Runtime tested with WH.
